cryptography/bls12_381/Cargo.toml

```diff
@@ -13,7 +13,7 @@ repository = { workspace = true }
 [dependencies]
 rayon = { workspace = true }
 
-blst = { version = "0.3", default-features = false }
+blst = { version = "0.3", default-features = false, features = ["no-threads"] }
```

Comment:
This would usually be controlled by the threadpool, but since its not been integrated, I just turn off all threads